### PR TITLE
Fix MirrorPercent in description snake_case to camelCase.

### PR DESCRIPTION
**Pull request by Taction**
For istio/istio#32884

### CODE REVIEW

File: networking/v1beta1/virtual_service.proto [hunk 1]
```diff
@@ -624,8 +624,8 @@ message HTTPRoute {
   Destination mirror = 9;
 
   // Percentage of the traffic to be mirrored by the `mirror` field.
-  // Use of integer `mirror_percent` value is deprecated. Use the
-  // double `mirror_percentage` field instead
+  // Use of integer `MirrorPercent` value is deprecated. Use the
```
Comment (linsun) [inline networking/v1beta1/virtual_service.proto:627]: should it be lower case m - `mirrorPercentage`?

**Comment by linsun** [comment]
FYI, you want to fix the alpha version as well.

File: networking/v1beta1/virtual_service.proto (hunk 1 above)
Comment (Taction) [reply networking/v1beta1/virtual_service.proto]: Fixed, thank you for pointing out that.

**Comment by Taction** [comment]
> FYI, you want to fix the alpha version as well.

I am not sure what to do for it, any suggestions? Thank you.

**Comment by craigbox** [comment]
> I am not sure what to do for it, any suggestions? Thank you.

Thanks for doing this, and jumping through all the hoops!

Lin is asking you to do everything that you've done in `networking/v1beta1` to `networking/v1alpha3` also. 🙂

**Comment by Taction** [comment]
Thank you. Fixed.

**Review by hzxuzhonghu — APPROVED** [review]

**Comment by hzxuzhonghu** [comment]
It can be closed, as it is hidden

**Pull request closed without merging**